### PR TITLE
hayt please only say hi once a day

### PR DESCRIPTION
**Pull request by justinwoo**
\<Muta_work> Does it hurt you to write PR descriptions?
\<justinw2> yes

### CODE REVIEW

**Comment by skalnik** [comment]
NO DESCRIPTION PROVIDED!

.vote no

**Comment by sshirokov** [comment]
> NO DESCRIPTION PROVIDED!
> 
> .vote no

FILL IT IN @justinwoo WHAT KIND OF PLACE DO YOU THINK I'M RUNNING HERE?

File: scripts/morning.coffee [hunk 1]
```diff
@@ -10,12 +10,15 @@
 # Commands:
 #   Hello or Good Day make hubot say hello to you back
 #   Good Morning makes hubot say good morning to you back
+ONE_DAY = 86400000
+last_time_i_wouldnt_shut_the_fuck_up = 0
```
Comment (skalnik) [inline scripts/morning.coffee]: Isn't javascript all about dat camel case?

Comment (justinwoo) [inline scripts/morning.coffee]: aren't we writing :coffee: :hankey: ?

Comment (sshirokov) [inline scripts/morning.coffee]: I think this symbol could use a few more syllables to be truly descriptive.

Comment (skalnik) [inline scripts/morning.coffee]: `thatOneTimeWhenISaidSomethingFuckingStupidAndWouldNotShutTheFuckUp = 0`

Comment (sshirokov) [inline scripts/morning.coffee]: :+1: Please integrate this critical change from the Review Board

**Pull request merged**